specification/examples/model-walkthrough_smdl.json

```diff
@@ -0,0 +1,49 @@
+{
+  "Name": "Simon IvC",
+  "BIDSModelVersion": "1.0.0",
+  "Input": {"subject": ["01", "02", "03"]},
```

Comment:
we probably wanna specify the task, no?

Reply:
I know but but probably specifying the task is still a fairly common used case to have it in there, no?